### PR TITLE
NIDM-experiment updates form Bay area meeting

### PR DESCRIPTION
**Pull request by cmaumet**
This pull request includes the changes made to NIDM-experiment at the Bay area meeting.

### CODE REVIEW

**Comment by cmaumet** [comment]
This is just a first pass at entering in owl the terms we discussed during the meeting (cf. [draft UML chart](https://drive.google.com/file/d/0ByYytkxqR-v0VktsVUllRTZaSE0/view?usp=sharing)). 

Please feel free to update the pull request to add more updates.

**Comment by cmaumet** [comment]
Preview of the spec: [here](http://htmlpreview.github.io/?https://raw.githubusercontent.com/incf-nidash/nidm/047a02eca856a8cee5f94c4006deecec50f2f42d/doc/content/specs/nidm-experiment_dev.html)

**Comment by nicholsn** [comment]
This looks like a great start! Do you want to merge this now or shall we start discussing here?

File: nidm/nidm-experiment/terms/nidm-experiment.owl [hunk 1]
```diff
@@ -10,6 +10,7 @@
 @prefix xsp: <http://www.owl-ontologies.com/2005/08/07/xsp.owl#> .
 @prefix daml: <http://www.daml.org/2001/03/daml+oil#> .
 @prefix meta: <http://purl.obolibrary.org/obo/iao/dev/ontology-metadata.owl#> .
+@prefix nidm: <http://purl.org/nidash/nidm/experiment#> .
```
Comment (nicholsn) [inline nidm/nidm-experiment/terms/nidm-experiment.owl]: do we want to use the top level namespace here? 

@prefix nidm: <http://purl.org/nidash/nidm#> .

**Comment by cmaumet** [comment]
Let's discuss here if that's okay... I think this still need more updates...

**Comment by cmaumet** [comment]
New preview: [here](http://htmlpreview.github.io/?https://raw.githubusercontent.com/incf-nidash/nidm/a752265f5b8a9ce2b53b706cf605da546e7be07a/doc/content/specs/nidm-experiment_dev.html)

**Comment by cmaumet** [comment]
@nicholsn: we should also add your figures in each of the levels.

**Comment by cmaumet** [comment]
Note: we also need to update the "contributors" list.

**Comment by nicholst** [comment]
UML image is truncated on the right.

**Comment by khelm** [comment]
Hi Camille,

   On the plane back I created a draft dictionary of the terms that will
be used in NIDM-Experiment, but I didn't have the document below with
me to work off of.  Should I be adding all of the terms there too or
are we still discussing?

regards,
Karl


> New preview:
> [here](http://htmlpreview.github.io/?https://raw.githubusercontent.com/incf-nidash/nidm/a752265f5b8a9ce2b53b706cf605da546e7be07a/doc/content/specs/nidm-experiment_dev.html)
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/incf-nidash/nidm/pull/245#issuecomment-72117583


-- 
Karl Helmer, PhD
Athinoula A Martinos Center
for Biomedical Imaging
Massachusetts General Hospital
149 - 13th St Room 2301
Charlestown, MA 02129
(p) 617.726.8636
(f) 617.726.7422
helmer@nmr.mgh.harvard.edu





The information in this e-mail is intended only for the person to whom it is
addressed. If you believe this e-mail was sent to you in error and the e-mail
contains patient information, please contact the Partners Compliance HelpLine at
http://www.partners.org/complianceline . If the e-mail was sent to you in error
but does not contain patient information, please contact the sender and properly
dispose of the e-mail.

**Comment by cmaumet** [comment]
Hi @khelm. I think it would be great if you could add those terms to the `nidm-experiment.owl` file so that we can discuss the model further. Please let me know if I can help!

File: nidm/nidm-experiment/terms/nidm-experiment.owl (hunk 1 above)
Comment (cmaumet) [inline nidm/nidm-experiment/terms/nidm-experiment.owl]: @nicholsn: yes, I agree with this update. But as this is currently under discussion at #237, I would suggest that we merge this PR first and update the namespace when we close #237.

**Comment by cmaumet** [comment]
(Just rebased on master.)

I think that we should merge a first version (before it start diverging again from master) and open separate issues for focused discussions. 

Is that okay with you?

**Comment by nicholsn** [comment]
cool, thanks. sounds good to me

**Comment by cmaumet** [comment]
Issues open for further discussions:
- NIDM-experiment spec: UML chart is truncated #249
- NIDM-experiment spec: draft dictionary from DICOM / Dataset terms #250

**Pull request merged**